deployment/build-s3-dist.sh

```diff
@@ -13,7 +13,7 @@
 #
 
 # Important: CDK global version number
-cdk_version=1.96.0
+cdk_version=2.59.0
```

Comment:
I don't think this is necessary. You can either trust your package-lock/npm-shrinkwrap or fix the exact version in the package file. I'm not sure that installing a specific version over what might already be there is guaranteed to have a good outcome.